microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -540,8 +568,24 @@ In **Settings** > **Experiments**, this **Disable the deprecated 'Color format'
 For now, these controls enable you to disable the experiment and re-activate the **Color format** drop-down list.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 113.
-*  This checkbox is present in Microsoft Edge Stable 111.
+*  This checkbox is present in Microsoft Edge Canary 114.
+*  This checkbox is present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
+## Enable background page selector (e.g. for prerendering debugging)
+
+_new as of Microsoft Edge 114_
+
+Enables the background page selector (e.g. for prerendering debugging).
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali need text about experiment checkbox "Enable background page selector (e.g. for prerendering debugging)"

Are there pages about "background page selector" or "prerendering debugging"?
"background page selector":
![image](https://user-images.githubusercontent.com/45407486/232161521-56957113-489e-45ed-8b39-2b261e1bcf18.png)

Reply:
kept as-is